ui/src/App.tsx

```diff
@@ -15,6 +15,7 @@ import IdentitiesPage from './components/IdentitiesPage.jsx';
 import ClaimsPage from './components/ClaimsPage.jsx';
 
 import BlockchainService from './services/BlockchainService.js';
+import Login from './components/Login.jsx';
```

Comment:
the casing for the login filename is wrong--should be Login.jsx, not login.jsx

Reply:
@jkrinsky it's updated, you can review now.